src/analysis/scripts/export_connectivity.sh

```diff
@@ -98,7 +98,7 @@ function ec_export_destination_geojson() {
   ogr2ogr -f GeoJSON "${FILENAME}" \
           -t_srs EPSG:4326 \
           "PG:host=${NB_POSTGRESQL_HOST} dbname=${NB_POSTGRESQL_DB} user=${NB_POSTGRESQL_USER}" \
-          -sql "select *, geom_pt from ${EXPORT_TABLENAME}"
+          -sql "select *, geom_pt from ${EXPORT_TABLENAME} where not ST_IsEmpty(geom_pt)"
```

Comment:
I would expect the other `ec_export_table_geojson` function to fail at some point because of a null geometry, but I'm not sure its possible to filter on null geometries if you don't know what the geometry column title is. Maybe `--skipfailures` would work instead? Or a broad geom filter on the bounding box of the entire world using `clipsrc`?